scadalts-ui/src/store/graphicView/storeIsAlive.js

```diff
@@ -74,6 +76,7 @@ const storeIsAlive = {
 					.then((res) => {
 						dispatch('calculateStateIsAlive', res.data).then(() => {
 							resolve(res);
+							axios.get(state.webhookUrl);
```

Comment:
1. `axios.get(state.webhookUrl)` is after resolve. Are you sure that this works correctly? I've checked that the requests are send to specific url so that works but...

2. Are you interpreting the response form that request?

Reply:
At present it should be passive, like "knocking". It is preferred to close the connection on our side as well.

Reply:
It's not needed to interpret the response.
This request is intended for telling to the watchdog the browser is receiving data from Scada server.
The triggers to restart computer is when the watchdog stops receiving the ping